doc/faq.rst

```diff
@@ -65,28 +65,25 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
+What are the inclusion criteria for new algorithms ?
+-----------------------------------------------------
+Generally we only add well-established algorithms. A rule of thumb is at least
 3 years since publications, 200+ citations and wide use and usefullness. A
-technique that provides a clear-cut improvement (e.g. an enhanced  data
-structure or efficient approximation) on a widely-used method will also be
-considered for inclusion.
+technique that provides a clear-cut improvement (e.g. an enhanced data
+structure or efficient approximation or both) on a widely-used method will 
+also be considered for inclusion. In general we only accept algorithms that
+fit well within the current API of scikit-learn, that is a ``fit``,
+``predict/transform`` interface. On top of this it is suggested that the
```

Comment:
I think drop "On top of this", make a new paragraph and say "The contributor should support the algorithm's importance by providing ..."